docs-ref-conceptual/what-is-azure-cli.md

```diff
@@ -1,40 +1,60 @@
 ---
-title: What is Azure CLI?
-description: Overview of Azure CLI, a command-line interface tool designed to create and manage Azure resources now available in Windows, macOS, and Linux environments.
+title: What is the Azure CLI?
+description: Overview of the Azure CLI, a command-line tool designed to create and manage Azure resources available in Windows, macOS, Linux, and Docker containers.
 author: dbradish-microsoft
 ms.author: dbradish
 manager: barbkess
-ms.date: 01/30/2020
+ms.date: 06/04/2021
 ms.topic: overview
 ms.service: azure-cli
 ms.devlang: azurecli 
 ms.custom: devx-track-azurecli
 ---
-# What is Azure CLI
+# What is the Azure CLI
 
-The Azure command-line interface (Azure CLI) is a set of commands used to create and manage Azure resources.  The Azure CLI is available across Azure services and is designed to get you working quickly with Azure, with an emphasis on automation.
+The Azure CLI is a cross-platform command-line tool to connect to Azure and execute administrative commands on Azure resources. It allows the execution of commands through a terminal using interactive command-line prompts or a script.
 
-## Key Characteristics
```

Comment:
You shift gears here, and I love the info you provide. 
Add an H2?
## How it works